includes/multi-currency/Helpers/OrderMetaHelper.php

```diff
@@ -0,0 +1,443 @@
+<?php
+/**
+ * Class OrderMetaHelper
+ *
+ * @package WooCommerce\Payments\MultiCurrency
+ */
+
+namespace WCPay\MultiCurrency\Helpers;
+
+use WC_Payments_API_Client;
+use WC_Payments_Features;
+use WC_Payments_Utils;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that controls OrderMetaHelper functionality.
+ */
+class OrderMetaHelper {
+	/**
+	 * Client for making requests to the WooCommerce Payments API.
+	 *
+	 * @var WC_Payments_API_Client
+	 */
+	private $payments_api_client;
+
+	/**
+	 * Array of errors, if any.
+	 *
+	 * @var array
+	 */
+	private $errors = [];
+
+	/**
+	 * Constructor.
+	 *
+	 * @param WC_Payments_API_Client $payments_api_client Payments API client.
+	 */
+	public function __construct( WC_Payments_API_Client $payments_api_client ) {
+		$this->payments_api_client = $payments_api_client;
+		$this->add_actions();
+	}
+
+	/**
+	 * Adds the button to toggle the tool on and off under WooCommerce > Status > Tools.
+	 *
+	 * @param array $tools The current array of tools.
+	 *
+	 * @return array The modified tools array.
+	 */
+	public function add_tool_button( $tools ) {
+		/**
+		 * We are not able to look at the current status of the tool to dynamically change these labels. The tool runs after the labels are created.
+		 */
+		$tools['wcpay_mc_meta_helper_toggle'] = [
+			'name'     => __( 'Enable/Disable Multi-Currency Meta Helper', 'woocommerce-payments' ),
+			'button'   => __( 'Enable/Disable', 'woocommerce-payments' ),
```

Comment:
I think we can conditionally change the button text to show Enable and Disable depending on the tool status. This way it's not clear if the tool is enabled or not.